.eslintrc.js

```diff
@@ -38,76 +43,79 @@ module.exports = {
     {
       files: ['*.ts'],
       extends: ['@metamask/eslint-config-typescript'],
+      parserOptions: {
+        project: ['./tsconfig.packages.json'],
+      },
       rules: {
         // disabled due to incompatibility with Record<string, unknown>
         // See https://github.com/Microsoft/TypeScript/issues/15300#issuecomment-702872440
         '@typescript-eslint/consistent-type-definitions': 'off',
 
-        // Modified to include the 'ignoreRestSiblings' option
-        // TODO: Migrate this rule change back into `@metamask/eslint-config`
-        '@typescript-eslint/no-unused-vars': [
-          'error',
-          {
-            vars: 'all',
-            args: 'all',
-            argsIgnorePattern: '[_]+',
-            ignoreRestSiblings: true,
-          },
-        ],
+        // TODO: auto-fix breaks stuff
+        '@typescript-eslint/promise-function-async': 'off',
+
+        // TODO: re-enble most of these rules
+        '@typescript-eslint/await-thenable': 'warn',
```

Comment:
Nit: Thoughts on using `off` instead of `warn`?

Warnings can add a lot of clutter, and people rarely heed them. Not sure how I feel about it though. Maybe it'll pressure us to come back to this.

Reply:
Sounds reasonable to me! We can adjust it later if we need to